utils/validators/validators.go

```diff
@@ -0,0 +1,108 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package validators contains environment variables validator.
+package validators
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// MetricsResolutions contains standard Prometheus metrics resolutions.
+type MetricsResolutions struct {
+	HR time.Duration
+	MR time.Duration
+	LR time.Duration
+}
+
+// EnvSettings contains PMM Server settings.
+type EnvSettings struct {
+	DisableUpdates     bool
+	DisableTelemetry   bool
+	MetricsResolutions MetricsResolutions
+	DataRetention      time.Duration
+}
+
+// EnvVarValidator validates given environment variables.
+// Returns two lists with errors and warnings.
+func EnvVarValidator(envs []string) (envSettings EnvSettings, errs []error, warns []string) {
+	for _, env := range envs {
+		p := strings.SplitN(env, "=", 2)
+		if len(p) != 2 {
+			errs = append(errs, fmt.Errorf("failed to parse environment variable %q", env))
+			continue
+		}
+
+		k, v := strings.ToUpper(p[0]), strings.ToLower(p[1])
+		switch k {
+		case "PATH", "HOSTNAME", "TERM", "HOME":
```

Comment:
Please a comment to that branch explaining why we skip them.

Reply:
Added.

Reply:
```
time="2020-02-21T06:32:10Z" level=warning msg="unknown environment variable \"PWD=/opt\"" component=pmm-managed-init
time="2020-02-21T06:32:10Z" level=warning msg="unknown environment variable \"SHLVL=1\"" component=pmm-managed-init
time="2020-02-21T06:32:10Z" level=warning msg="unknown environment variable \"_=/usr/sbin/pmm-managed-init\"" component=pmm-managed-init
```

Looks like we also need to add them.